prowler/providers/aws/services/dynamodb/dynamodb_service.py

```diff
@@ -120,6 +123,37 @@ def _get_resource_policy(self):
                 f"{regional_client.region} -- {error.__class__.__name__}[{error.__traceback__.tb_lineno}]: {error}"
             )
 
+    def _describe_autoscaling(self):
+        logger.info("DynamoDB - Describing Auto Scaling...")
+        try:
+            for table in self.tables.values():
+                if table.billing_mode == "PROVISIONED":
+                    application_autoscaling_client = self.session.client(
+                        "application-autoscaling", region_name=table.region
+                    )
+                    read_response = (
+                        application_autoscaling_client.describe_scalable_targets(
+                            ServiceNamespace="dynamodb",
+                            ResourceIds=[f"table/{table.name}"],
+                            ScalableDimension="dynamodb:table:ReadCapacityUnits",
+                        )
+                    )
+                    if read_response["ScalableTargets"]:
+                        table.read_autoscaling = True
+                    write_response = (
+                        application_autoscaling_client.describe_scalable_targets(
+                            ServiceNamespace="dynamodb",
+                            ResourceIds=[f"table/{table.name}"],
+                            ScalableDimension="dynamodb:table:WriteCapacityUnits",
+                        )
+                    )
+                    if write_response["ScalableTargets"]:
+                        table.write_autoscaling = True
+        except Exception as error:
+            logger.error(
+                f"{error.__class__.__name__}:{error.__traceback__.tb_lineno} -- {error}"
+            )
+
```

Comment:
Please, as you are using another client, put this function in the `autoscaling` service in a new class.